ecommerce/urls.py

```diff
@@ -74,6 +74,7 @@ def handler403(_, exception):  # pylint: disable=unused-argument
     url(r'^credit/', include(('ecommerce.credit.urls', 'credit'))),
     url(r'^coupons/', include(('ecommerce.coupons.urls', 'coupons'))),
     url(r'^enterprise/', include(('ecommerce.enterprise.urls', 'enterprise'))),
+    url(r'^enterprise-coupon/', include(('ecommerce.enterprise_coupons.urls', 'enterprise_coupons'))),
```

Comment:
This view would manage all types of coupons, although for now we only plan to manage Enterprise coupons. I think it would be appropriate to call it 'coupon-view' or so. @daviidco